src/main/java/forestry/core/climate/ClimateTransformer.java

```diff
@@ -117,6 +117,7 @@ public void readFromNBT(NBTTagCompound nbt) {
 	@Override
 	public IClimateManipulatorBuilder createManipulator(ClimateType type) {
 		return new ClimateManipulator.Builder()
+				.setAllowBackwards()
```

Comment:
This breaks the humidity implementation of the habitat former because the humidity should only be controlled through the fluids and this allows to lower the humidity by consuming water which makes no sense. There already is a `setAllowBackwards()` in `TileHabitatFormer#updateTemperature` for the regulation of the temperature.

Reply:
Yep, my mistake.